code/game/machinery/computer/cloning.dm

```diff
@@ -358,35 +358,31 @@
 		return
 	if(scan_brain && !can_brainscan())
 		return
-	if((isnull(subject)) || (!(ishuman(subject))) || (!subject.dna) || (NO_SCAN in subject.dna.species.species_traits))
+	if(isnull(subject) || (!(ishuman(subject))) || (!subject.dna))
 		scantemp = "<span class=\"bad\">Error: Unable to locate valid genetic data.</span>"
 		SSnanoui.update_uis(src)
 		return
 	if(subject.get_int_organ(/obj/item/organ/internal/brain))
 		var/obj/item/organ/internal/brain/Brn = subject.get_int_organ(/obj/item/organ/internal/brain)
 		if(istype(Brn))
 			if(NO_SCAN in Brn.dna.species.species_traits)
-				scantemp = "<span class=\"bad\">Error: Subject's brain is incompatible.</span>"
+				scantemp = "<span class=\"bad\">Error: [subject.dna.species.name_plural] are not scannable.</span>"
 				SSnanoui.update_uis(src)
 				return
 	if(!subject.get_int_organ(/obj/item/organ/internal/brain))
-		scantemp = "<span class=\"bad\">Error: No signs of intelligence detected.</span>"
+		scantemp = "<span class=\"bad\">Error: No brain detected in subject.</span>"
 		SSnanoui.update_uis(src)
 		return
 	if(subject.suiciding)
-		scantemp = "<span class=\"bad\">Error: Subject's brain is not responding to scanning stimuli.</span>"
+		scantemp = "<span class=\"bad\">Error: Subject has committed suicide and is not scannable.</span>"
 		SSnanoui.update_uis(src)
 		return
 	if((!subject.ckey) || (!subject.client))
-		scantemp = "<span class=\"bad\">Error: Mental interface failure.</span>"
+		scantemp = "<span class=\"bad\">Error: Subject's brain is not responding. Further attempts may succeed.</span>"
```

Comment:
Perhaps 'further attempts after a short delay may succeed', so they don't just hit it 3 times in 3 seconds, then give up, before the ghost has a chance to re-enter.

Reply:
Good point.  Only reason I used this message is because it matches what the defib says when the ghost isn't in the body.